src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -86,6 +86,24 @@ public int GetArrayLength()
             return _parent.GetArrayLength(_idx);
         }
 
+        /// <summary>
+        ///   Get the number of properties contained within the current object value.
+        /// </summary>
+        /// <returns></returns>
+        /// <returns>The number of properties contained within the current object value.</returns>
+        /// <exception cref="InvalidOperationException">
+        ///   This value's <see cref="ValueKind"/> is not <see cref="JsonValueKind.Array"/>.
```

Comment:
I think the message here should say `JsonValueKind.Object`

Reply:
```suggestion
        ///   This value's <see cref="ValueKind"/> is not <see cref="JsonValueKind.Object"/>.
```